src/translations/pages/contact.ts

```diff
@@ -954,5 +954,21 @@ export const Contact = {
           ),
       },
     },
+    modal: {
+      open: {
+        ariaLabel: _(
+          'Klikk for å åpne modal',
```

Comment:
I think we should have a more descriptive label here. For example something like "More information about ${inputName}".

Reply:
Will update👍